### PR TITLE
Setup Azure Alert for Azure is down

### PR DESCRIPTION
**Pull request by jherrflexion**
# Setup Azure Alert for Azure is down

Added terraform for creating Azure Alert for when Azure is down. 

## Issue

https://github.com/CDCgov/trusted-intermediary/issues/1397

## Checklist

**Note**: You may remove items that are not applicable

### CODE REVIEW

File: operations/template/variables.tf [hunk 1]
```diff
@@ -19,3 +19,8 @@ variable "alert_slack_email" {
   nullable  = false
   sensitive = true
 }
+
+variable "service_health_locations" {
+  type    = string
+  default = "global"
+}
```
Comment (halprin) [inline operations/template/variables.tf:26]: Do we need this?  Do we actually customize this to anything?  I feel we could just hardcode `"global"` in the `azurerm_monitor_activity_log_alert`.

Comment (pluckyswan) [reply operations/template/variables.tf]: Removed variable for now as its only used in this file. If more instances pop up, will refactor later.

File: operations/template/alert.tf [hunk 2]
```diff
@@ -28,6 +28,47 @@ resource "azurerm_monitor_action_group" "notify_slack_email" {
   }
 }
 
+resource "azurerm_monitor_activity_log_alert" "azure_service_health_alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdcti-${var.environment}-azure-status-alert"
+  location            = "global"
```
Comment (halprin) [inline operations/template/alert.tf:34]: This should be `data.azurerm_resource_group.group.location`?  For every other resource that asks for a `resource_group_name` and `location`, we've put `data.azurerm_resource_group.group.location` for the `location`.

**Review by halprin — APPROVED** [review]
The System is Down

**Pull request merged**